### ISSUE TITLE
Enhancement requests

### ISSUE THREAD

**Issue by REMC19** [Swing-team/pendulums-web-client#113]
Allow user to specify their own from and to dates in the activities list (or give options longer then last 3 months).
Allow total times to be shown as decimal versions as a preference (i.e. 7.5 hours rather than 7hrs 30 minutes, or 100.3 rather than 100 hrs 30 minutes).

**Comment by mohammadrafigh** [comment]
- You can already specify custom date range by clicking on the calendar.
- Your second suggestion is not an standard way to read times but I will put it into our backlog with lower priority.

Thank you for your suggestions.

**Comment by REMC19** [comment]
I don't think you can just click on the dates, I tried setting the first one, then going to another month clicking on that date - it just set both of the dates to the same (2nd) date.
If you could detail how you are doing it, it would be helpful - because I couldn't see what I was doing wrong!

**Comment by mohammadrafigh** [comment]
What is your platform? Desktop/Laptop or Android? So I can send youa screen
record on how to do that.

**Comment by REMC19** [comment]
Desktop

**Comment by mohammadrafigh** [comment]
Ok, I don't have access to my laptop till tomorrow. I will send an screen
recording tomorrow.

**Comment by mohammadrafigh** [comment]
Here it is

[2022-12-09 10-50-27.webm](https://user-images.githubusercontent.com/6737508/206650332-d4a070f5-a1cf-443e-807f-b59eded05555.webm)
